icu4j/main/classes/core/src/com/ibm/icu/impl/units/UnitPreferences.java

```diff
@@ -0,0 +1,210 @@
+// © 2020 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+
+package com.ibm.icu.impl.units;
+
+import com.ibm.icu.impl.ICUData;
+import com.ibm.icu.impl.ICUResourceBundle;
+import com.ibm.icu.impl.UResource;
+import com.ibm.icu.util.UResourceBundle;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.TreeMap;
```

Comment:
HashMap, unless you need it to be sorted

Reply:
I use only HashMap